timemachine/ff/handlers/nonbonded.py

```diff
@@ -187,26 +189,29 @@ def bond_smirks_matches(mol, smirks_list):
     * Order within each smirks pattern matters
         For example, "[#6:1]~[#1:2]" and "[#1:1]~[#6:2]" will match atom pairs in the opposite order
     """
-    oemol = convert_to_oe(mol)
-    AromaticityModel.assign(oemol)
+    if not mol.HasProp(AM1_SMIRK_MATCH_CACHE):
+        oemol = convert_to_oe(mol)
+        AromaticityModel.assign(oemol)
```

Comment:
Another possibility to remove the OpenEye dependence of this function might be to compute the list of bond smirks matches using RDKit -- but I don't immediately remember if there are important RDKit / OpenEye differences in how these SMIRKS are interpreted.

(In some corner cases, identical SMIRKS strings can have different meanings between RDKit and OpenEye.)

Reply:
I had considered this, though since we use OpenEye already for Quacpac it seemed natural to keep this. Plus exactly what you said about corner cases.